tests/test_media_multipart.py

```diff
@@ -849,3 +850,14 @@ async def deserialize_async(self, stream, content_type, content_length):
 
     assert resp.status_code == 200
     assert resp.json == ['', '0x48']
+
+
+def test_multipart_parse_options_default_handlers_unique():
+    parse_options_one = MultipartParseOptions()
+    parse_options_two = MultipartParseOptions()
+
+    parse_options_one.media_handlers.pop(falcon.MEDIA_JSON)
+
+    assert id(parse_options_one.media_handlers) != id(parse_options_two.media_handlers)
```

Comment:
This works, but it would be more idiomatic to simply write
```python
    assert parse_options_one.media_handlers is not parse_options_two.media_handlers
```